go.mod

```diff
@@ -0,0 +1,14 @@
+module go-dfms-gateway
+
+go 1.14
+
+require (
+	github.com/gabriel-vasile/mimetype v1.1.1
+	github.com/ipfs/go-ipfs-files v0.0.8
+	github.com/ipfs/go-log v1.0.4
+	github.com/proximax-storage/go-xpx-dfms-api v0.3.0
+	github.com/proximax-storage/go-xpx-dfms-api-http v0.3.0
+	github.com/proximax-storage/go-xpx-dfms-drive v0.2.1-0.20200128151550-5d6565ba489f
+	github.com/stretchr/testify v1.6.1
+	github.com/valyala/fasthttp v1.15.1
```

Comment:
I think you will be also interested in: https://github.com/gofiber/fiber.
It is a fasthttp based framework for HTTP server.

Reply:
Ok, do you think we should use it? I think just fasthttp is enough.

Reply:
No, just for your information